public/request/auth/send-password-reset-email.php

```diff
@@ -1,12 +1,17 @@
 <?php
 
+use App\Models\User;
 use Illuminate\Support\Arr;
 use Illuminate\Support\Facades\Validator;
 
 $input = Validator::validate(Arr::wrap(request()->post()), [
     'username' => 'required',
 ]);
 
-RequestPasswordReset($input['username']);
+$targetUser = User::firstWhere('User', $input['username']);
+
+if (!$targetUser->isBanned()) {
```

Comment:
Need a null check here. If you try to reset the password for a non-existant username, you get an error.

Reply:
Good point - added in latest.